src/julia_threads.h

```diff
@@ -243,16 +243,8 @@ typedef struct _jl_task_t {
     jl_value_t *result;
     jl_value_t *scope;
     jl_function_t *start;
-    _Atomic(uint8_t) _state;
-    uint8_t sticky; // record whether this Task can be migrated to a new thread
-    uint16_t priority;
-    _Atomic(uint8_t) _isexception; // set if `result` is an exception to throw or that we exited with
-    uint8_t pad0[3];
-    // === 64 bytes (cache line)
+    // === 56 bytes
     uint64_t rngState[JL_RNG_SIZE];
```

Comment:
As a down side, this now sits "across" a cache line -- which may not be a problem, just thought it should be mentioned explicitly :-)

Reply:
Interestingly the `rngState` actually is split in "4+1" (xoshiro 4 + splitmix 1). If this was flipped (to 1+4) it would align more nicely with the cacheline... but since all of the `rngState` is accessed at once it probably won't matter either way.